javadoc-parser-documentation/src/main/asciidoc/documentation.adoc

```diff
@@ -32,12 +36,54 @@ JavaDoc javaDoc = parser.parse(inputString); //<1>
 List<AuthorTag> authors = javaDoc.getTags(AuthorTag.class); // <2>
 
 AuthorTag authorTag = authors.get(0);
-String authorName = authorTag.getAuthorName(); // <3>
+String authorName = authorTag.getNameText(); // <3>
 ----
 <1> Parsing input string into a java object
 <2> Reading a specific tag from `JavaDoc` object
 <3> Getting information from the current tag
 
+== Registered tags
+The `JavaDocParserBuilder` provides three methods to register a basic set of tags.
+
+`.withoutTags()`::
+No Tags are registered by default.
+Each one has to be added manually.
+
+`.withStandardJavadocTags()`::
+The following tags are registered according to the link:https://docs.oracle.com/en/java/javase/18/docs/specs/javadoc/doc-comment-spec.html[Javadoc documentation]
+* `@author`
+* `@deprecated`
+* `@exception`
+* `@hidden`
+* `@param`
+* `@provides`
+* `@return`
+* `@see`
+* `@serialData`
+* `@serialField`
+* `@serial`
+* `@since`
+* `@throws`
+* `@uses`
+* `@version`
```

Comment:
This list not full. Must be:
```
@author
@code
@deprecated
@docRoot
@exception
@hidden
@index
@inheritDoc
@link
@linkplain
@literal
@param
@provides
@return
@see
@serial
@serialData
@serialField
@since
@snippet
@summary
@systemProperty
@throws
@uses
@value
@version
```

Reply:
The documentation was updated that only block tags are supported. Inline tags are handled in a separate manner.